python/cudf/cudf_pandas_tests/third_party_integration_tests/tests/test_catboost.py

```diff
@@ -84,24 +95,26 @@ def test_catboost_with_pool_and_dataframe(regression_data):
 def test_catboost_with_pool_and_numpy(regression_data):
     X, y = regression_data
     train_pool = Pool(X.values, y.values)
-    model = CatBoostRegressor(iterations=10, verbose=0)
+    model = CatBoostRegressor(iterations=50, verbose=0)
     model.fit(train_pool)
     predictions = model.predict(X.values)
     return predictions
 
 
 def test_catboost_with_categorical_features():
     data = {
-        "numerical_feature": rng.standard_normal(100),
-        "categorical_feature": rng.choice(["A", "B", "C"], size=100),
-        "target": rng.integers(0, 2, size=100),
+        "fea_1": rng.standard_normal(1000),
+        "fea_2": rng.choice([0, 1, 2, 3], size=1000),
+        "fea_3": rng.choice(["A", "B", "C"], size=1000),
```

Comment:
```suggestion
        "cat_feat_int_dtype": rng.choice([0, 1, 2, 3], size=1000),
        "cat_feat_categorical_dtype": rng.choice(["A", "B", "C"], size=1000),
```

Would you consider a change like this, or a code comment capturing this information if you'd prefer? Something really important is happening here...

* `fea_2` is testing "using an `int` column as a categorical feature works"
* `fea_3` is testing "using a `categorical` column as a categorical feature works"

*(based on https://github.com/rapidsai/cudf/pull/17931#discussion_r1947207063)*

But I don't think that important difference would be obvious to someone looking at this test for the first time who isn't familiar with these boosting libraries.